src/OpenTelemetry.Instrumentation.AspNet/README.md

```diff
@@ -189,7 +189,13 @@ this.tracerProvider = Sdk.CreateTracerProviderBuilder()
         };
         o.EnrichWithException = (activity, exception) =>
         {
-            activity.SetTag("exceptionType", exception.GetType().ToString());
+            // If there is an inner exception, capture its details as well
+            if (exception.InnerException != null)
+            {
+                activity.SetTag("exception.inner.type", exception.InnerException.GetType().FullName);
+                activity.SetTag("exception.inner.message", exception.InnerException.Message);
+                activity.SetTag("exception.inner.stacktrace", exception.InnerException.StackTrace);
```

Comment:
not sure if this is potentially confusing.. given `stacktrace` is filled with `ToString()` by the otel core api (and in .net runtime too due to recent addition)..
Also, the inner could have further inner?

Reply:
I guess one level of depth is sufficient for the example. Will try to find another example if this one is confusing for you.

Reply:
It is not confusing to me, really. What I meant to share is that - quite often, people just copy the code as-is, ignoring the fact that this is merely an example of what can be done using the Enrich mechanism!

Maybe use https://learn.microsoft.com/en-us/dotnet/api/system.exception.source?view=net-8.0#system-exception-source instead?

Reply:
I'm sorry, I didn't mean that it was confusing for you, but that you had the feeling that it could be confusing, in general. Thank you for the suggestion, I used it.